index.html

```diff
@@ -166,6 +166,11 @@
           <label class="form-check-label" for="pixel-diff-selector">[EXPERIMENTAL] Use the Pixel difference resolution selector (reloads player)</label>
         </div>
 
+        <div class="form-check">
```

Comment:
can this be moved above the experimental items?

Reply:
Sure thing 👍